docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "069a3ce3",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# %%capture \n",
```

Comment:
Can you try to closely mimic the format of our other existing tutorial notebooks for this cell?  Happy to discuss it in chat, but the basic idea is this cell needs to be structured in a very particular way to make sure the installation of dependencies happens properly when docs.cleanlab.ai executes this notebook during CI, when the user runs this notebook on google colab, or when they run it locally.  For now, I'd just take a stab at making this cell look as similar to the installation cell in our existing notebooks, and then I'll walk you through what's still missing next time we meet.